Cargo.toml

```diff
@@ -9,8 +9,9 @@ edition = "2021"
 anyhow = "1.0"
 camino = "1.0"
 clap = { version = "3.0", features = ["derive"] }
-log = "0.4"
-tokio = { version = "1", features = ["full"] }
+tokio = { version = "1", features = ["fs", "io-util", "sync", "macros", "rt-multi-thread"] }
+tracing = "0.1.35"
+tracing-subscriber = { version = "0.3.11", features = ["env-filter"] }
```

Comment:
since `log` wasn't being used, I replaced it with the [`tracing`][0] crate, which is compatible with `log` but has many other nice features. [More info][1]

[0]: https://docs.rs/tracing/latest/tracing/#for-log-users
[1]: https://fasterthanli.me/articles/request-coalescing-in-async-rust#a-bit-of-tracing

Reply:
Hum, `tracing` is really nice! Just to make sure: so I got the namespace wrong? I thought I was using `log` here: 

https://github.com/cuducos/grrs/blob/e331409536f2a7b3d7a0129df8798e0b85a3f9f6/src/main.rs#L27-L30

Reply:
Indeed, I ended up removing that line, so that's why `log` was removed as a dep... But you're right, you can still keep `log` around and use `warn!`, or use the `tracing` equivalent.

If this was a lib I would tend to suggest `log` (because `tracing` can interact with it) to avoid forcing a specific solution for your dependents. But since this is an application you can use what you think is more featureful/useful (and that's where I usually use more opinionated crates for error formatting or logging, for example)

Reply:
Thank you! Tons of learnings : )